prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -499,4 +506,5 @@ class Image(BaseModel):
 
 class EbsEncryptionByDefault(BaseModel):
     status: bool
+    volumes: bool
```

Comment:
Would it not be better to save a map of regions to know if there are volumes in there? We could need it.

Like:
`self.volumes_in_region = {AWS_REGION_1: True, AWS_REGION_2: False}`

Reply:
I think in that case is not necessary since we have an encryption configuration per each region.